acceptance/tests/resource/service/service_enable_linux.rb

```diff
@@ -0,0 +1,93 @@
+test_name 'SysV and Systemd Service Provider Validation'
+
+confine :except, :platform => 'windows'
+# test not yet written for launchd platform(s)
+confine :except, :platform => /osx/
+confine :except, :platform => 'solaris'
+# upstart provider tested in ticket_14297_handle_upstart.rb
+confine :except, :platform => /ubuntu-[a-u]/
+
+package_name = {'el'     => 'httpd',   'centos' => 'httpd', 'sles' => 'httpd', 'fedora' => 'httpd',
+                'ubuntu' => 'apache2', 'debian' => 'apache2'}
+
+init_script_el7   = "/usr/lib/systemd/system/httpd.service"
+symlink_el7       = "/etc/systemd/system/multi-user.target.wants/httpd.service"
+
+start_runlevels = ["2", "3", "4", "5"]
+kill_runlevels = ["0", "1", "6"]
+
+agents.each do |agent|
+  platform = agent.platform.split('-')[0]
```

Comment:
Does the `platform` always contain a dash? Is there a beaker helper to return just the platform name minus the arch?